documentation/modules/post/networking/gather/enum_cisco.md

```diff
@@ -111,7 +111,7 @@ resource (cisco.rb)> run
 [*] Command shell session 1 opened (111.111.1.111:41839 -> 222.222.2.222:22) at 2019-07-21 16:24:02 -0400
```

Comment:
Similar problem here. We note that `cisco.rb` is being used as some sort of resource file but I don't see any documentation on how we are actually loading that resource file or even if it is needed to run this. Also even after you load the module the prompt isn't changing like it would in real life, so this needs to be altered.